unitylibs/core/workflow/workflow-firefly/action-binder.js

```diff
@@ -164,68 +176,9 @@ export default class ActionBinder {
     if (execute) await execute();
   }
 
-  async fetchAutoComplete(fetchType = 'default') {
-    try {
-      if (!this.serviceHandler) await this.loadServiceHandler();
-      this.maxResults = fetchType === 'refresh' ? this.maxResults * 2 : 12;
-      if (fetchType !== 'refresh' && this.query) sendAnalyticsEvent(new CustomEvent('promptInput', { detail: { query: this.query } }));
-      const payload = {
-        query: this.query,
-        targetProduct: this.apiConfig.productName,
-        maxResults: this.maxResults,
-      };
-      const response = await this.serviceHandler.postCallToService(
-        this.apiConfig.expressEndpoint.autoComplete,
-        { body: JSON.stringify(payload) },
-      );
-      if (response?.completions) {
-        this.suggestion = fetchType === 'refresh'
-          ? response.completions.slice(this.maxResults / 2)
-          : response.completions;
-        this.displaySuggestions();
-        this.inputField.focus();
-      }
-    } catch (err) {
-      console.error('Error fetching autocomplete suggestions:', err);
-    }
-  }
-
-  async refreshSuggestions() {
-    if (this.suggestion.length) {
-      this.displaySuggestions();
-      this.inputField.focus();
-      return;
-    }
-    await this.fetchAutoComplete('refresh');
-  }
-
-  displaySuggestions() {
-    this.clearDropdown();
-    this.toggleDefaultItems(false);
-    const dynamicHeader = this.createDynamicHeader();
-    this.dropdown.prepend(dynamicHeader);
-    if (this.suggestion.length === 0) {
-      this.showEmptyState(dynamicHeader);
-      this.scrRead.textContent = this.workflowCfg.placeholder['placeholder-no-suggestions'];
-    } else {
-      const suggestionsToAdd = this.suggestion.splice(0, 3);
-      this.addSuggestionItems(suggestionsToAdd, dynamicHeader);
-      this.scrRead.textContent = `${suggestionsToAdd.length} suggestions ${suggestionsToAdd.reverse().toString()} available. Use up and down arrows to navigate`;
-    }
-    this.showDropdown();
-    this.initActionListeners();
-    this.addAccessibility();
-  }
-
-  async triggerSurprise() {
-    const prompts = this.workflowCfg?.supportedTexts?.prompt || [];
-    if (!prompts.length) return;
-    this.query = prompts[Math.floor(Math.random() * prompts.length)];
-    await this.generateContent();
-  }
-
```

Comment:
Is autocomplete scheduled to go in at a later date? If so, do we want to keep it in code?